{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/settings/dist.py

```diff
@@ -35,14 +39,18 @@
             'class': 'logging.StreamHandler',
             'formatter': 'verbose'
         },
+        'json': {
```

Comment:
I'm nitpicking, but I'd name it `console_json` to indicate the output stream too. Just by the name I have no idea where the json would be outputted.